src/main/java/br/com/openenade/api/modalidade/ModalidadeController.java

```diff
@@ -0,0 +1,20 @@
+package br.com.openenade.api.modalidade;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/modalidade")
```

Comment:
@Icaro-Lima please see how @javanlacerda is using the RequestMapping [value](https://github.com/OpenEnade/API/blob/cc2def1e88c2915bc16ac9d60de3eeec0100b61c/src/main/java/br/com/openenade/api/AnoController.java#L19).